library/include/ck/library/reference_tensor_operation/cpu/reference_layernorm.hpp

```diff
@@ -90,10 +90,11 @@ struct ReferenceLayernorm : public device::BaseOperator
 
             for(int m = 0; m < M; ++m)
             {
+                auto divisor = 1 / sqrt(var(m) + arg.epsilon_);
```

Comment:
I think this is dangerous,  I could not which is the type of `divisor`,  had better to use `static_cast<AccDataType>(1)` .   Also add `using ck::math::sqrt;`, so that `sqrt` is the one under our control

Reply:
Fix in 4522f7f8760dbe43aa2f1e2b3655cb3776c639a6